src/components/20-molecules/cookie-disclaimer/index.js

```diff
@@ -0,0 +1,103 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+const _global = window || global;
+
+class AXACookieDisclaimer extends LitElement {
+  static get tagName() {
+    return 'axa-cookie-disclaimer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    // Define properties and types
+    return {
+      onClick: { type: Function },
+      buttonname: { type: String },
+      title: { type: String },
+      variant: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.onClick = () => {};
+  }
+
+  firstUpdated() {
+    if (this.hasAccepted()) {
+      this.selfDestruction();
+    } else {
+      const resetStyle = document.createElement('style');
```

Comment:
Maybe here comment here to describe the problem. In this workaround

Reply:
Done